### DISCUSSION TITLE
Questions regarding connectivity values using Charm and Head Model Mesh

### DISCUSSION THREAD

**Discussion by jack1125happy** [simnibs/simnibs#319] [2. General]
Dear Colleagues,
I am writing to seek your professional opinions on two matters related to SimNIBS 4.0. Your input would be greatly appreciated.

1) Regarding the default connectivity values of the 15 tissues, how can I adjust the connectivity values when utilizing CHARM?

2) Where can the mesh details of the head model generated by SimNIBS 4.0 be found? e.g., mesh resolution, convergence results, etc.

Thank you very much for any advice.

Best regards,
Jack

**Comment by TorgeW** [comment]
I think you mean the conductivity?
You can adjust the conductivity values when running simulations. 
GUI:
https://simnibs.github.io/simnibs/build/html/documentation/gui.html#conductivities-window
Script:
https://simnibs.github.io/simnibs/build/html/documentation/sim_struct/cond.html


2.
https://iopscience.iop.org/article/10.1088/1741-2552/ab41ba
